src/components/util-browser.js

```diff
@@ -1,4 +1,5 @@
 var markoUID = window.$MUID || (window.$MUID = { i: 0 });
+
```

Comment:
Would you mind removing this space from the PR?

Reply:
done :)